tests/e2e/specs/gtag-events/classic-pages.test.js

```diff
@@ -74,6 +74,32 @@ test.describe( 'GTag events on classic pages', () => {
 		} );
 	} );
 
+	test( 'Add to cart quantity is sent on a single product page', async ( {
+		page,
+	} ) => {
+		const event = trackGtagEvent( page, 'add_to_cart' );
+
+		await page.goto( `?p=${ simpleProductID }` );
+
+		await page.locator( '.quantity input.qty' ).first().fill( '3' );
+
+		const addToCart = `.single_add_to_cart_button[value="${ simpleProductID }"]`;
+		const addToCartButton = await page.locator( addToCart ).first();
+
+		await addToCartButton.click();
```

Comment:
❔ What do you think adding a `page.reload()` for the classic shop as well so it would match the testing instruction in the PR description?

Reply:
Thanks for the review @ianlin 🙌 

The error with the product quantity on the single page wasn't triggered with a page reload in the same way the other one was. When the `added_to_cart` hook runs the quantity wasn't being added to the product object so for the test we just need to click the `Add to cart` button and then confirm the quantity is now present in the event. Apologies if that wasn't clear from the testing instructions.